armada/armada.py

```diff
@@ -59,16 +59,18 @@ def sync(self):
             protoc_chart = chartbuilder.get_helm_chart()
 
             # determine install or upgrade by examining known releases
+            LOG.info("RELEASE: %s", chart.release_name)
```

Comment:
Don't need these statements, they were for debugging (or can move to DEBUG) but I would just remove